setup.cfg

```diff
@@ -54,6 +54,10 @@ install_requires =
 python_requires = >=3.6
 include_package_data = True
 
+[options.extras_require]
```

Comment:
I'm not a big fan of using extras for this, but I stuck with a minimal change to the existing build / packaging setup.